src/components/Sheet/README.md

```diff
@@ -0,0 +1,203 @@
+
+---
+
+## Accessibility
+
+Sheets provide an opportunity to let merchants dig into more detail on their current task, or access information for their current task in a different way. Although merchants may be able to see content in the sheet and the main page content at the same time, they should only be expected to interact with one or the other at any given time.
+
+### Keyboard support
+
+- Use the `onClose` prop so that the sheet can be closed with the <kbd>esc</kbd> key as well as with button-based controls
+- Use a button to open the sheet
+- When the sheet opens, move focus to it so merchants who rely on the keyboard and screen readers can access it
+- When the sheet closes, move focus back to the button that launched it
+
+---
+
+## Best practices
+
+The sheet component should:
+
+- Include a heading that summarizes the actions and information in the sheet, for example, More filters
+- Be openable through clear actions, like a link or button
+- Be close-able through clear actions, like Done, the [X] button, and the esc key
+- Include information and actions contextual to the current task
+- Not block merchants from completing their task, like a modal would
+- Not open from within another sheet (only one sheet can be open at a time)
+- Preserve its state—the settings and actions won’t reset when it’s closed
+
+---
+
+## Examples
+
+### Basic sheet
+
+<!-- example-for: web -->
+
+Use as the default option for a sheet.
+
+```jsx
+class SheetExample extends React.Component {
+  state = {
+    sheetActive: false,
+    newsletter: false,
+    email: '',
+  };
+
+  render() {
+    const theme = {
+      colors: {
+        topBar: {
+          background: '#357997',
+        },
+      },
+      logo: {
+        width: 124,
+        topBarSource:
+          'https://cdn.shopify.com/s/files/1/0446/6937/files/jaded-pixel-logo-color.svg?6215648040070010999',
+        contextualSaveBarSource:
+          'https://cdn.shopify.com/s/files/1/0446/6937/files/jaded-pixel-logo-gray.svg?6215648040070010999',
+        url: 'http://jadedpixel.com',
+        accessibilityLabel: 'Jaded Pixel',
+      },
+    };
+
+    const {sheetActive, newsletter, email} = this.state;
+
+    return (
+      <AppProvider theme={theme}>
+        <Frame topBar={<TopBar />}>
+          <Page title="Sheet">
+            <Card sectioned>
+              <Button onClick={this.handleToggleSheet}>
+                {sheetActive ? 'Close sheet' : 'Open sheet'}
+              </Button>
+            </Card>
+            <Sheet open={sheetActive} onClose={this.handleCloseSheet}>
+              <div
+                style={{
+                  display: 'flex',
+                  flexDirection: 'column',
+                  height: '100%',
+                }}
+              >
+                <div
+                  style={{
+                    alignItems: 'center',
```

Comment:
Doens't look like `alignItems` is being accounted for correctly: 
![https://screenshot.click/02-12-jbpj3-1pj07.jpg](https://screenshot.click/02-12-jbpj3-1pj07.jpg)

Reply:
|:tophat:|
|-|
|![Screen Shot 2019-05-03 at 8 51 55 AM](https://user-images.githubusercontent.com/1403188/57138489-c8959b00-6d80-11e9-896e-39d0c9fa951d.png)|

LGTM